apps/demo/src/Views/Ui/index.tsx

```diff
@@ -52,6 +57,15 @@ const componentList = [
     title: "buttons.export",
     component: <ExportButton />,
   },
+  {
+    key: 6,
+    title: "files.table",
+    component: (
+      <FilesTable
+        files={[{ filename: "myfile", uploadedBy: "Manish", uploadedAt: 312 }]}
```

Comment:
It'd be nice to have realistic data.
```suggestion
        files={[{ filename: "myfile.pdf", uploadedBy: <userObject>, uploadedAt: <timestamp> }]}
```